### PR TITLE
SEO-179239-Redirect-Links-TS

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on links with redirects.

Regards,
Beryl.

### CODE REVIEW

File: ej2-javascript/datepicker/ts/getting-started.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ domainurl: ##DomainURL##
 
 # Getting started in ##Platform_Name## Datepicker control
 
-This section briefly explains how to create a simple [JavaScript DatePicker](https://www.syncfusion.com/javascript-controls/js-datepicker) component and configure its available functionalities in TypeScript, using Essential JS 2 [quickstart](https://github.com/SyncfusionExamples/ej2-quickstart-webpack-) seed repository.
+This section briefly explains how to create a simple [JavaScript DatePicker](https://www.syncfusion.com/javascript-controls/js-datepicker) component and configure its available functionalities in TypeScript, using Essential JS 2 [quickstart](https://github.com/SyncfusionExamples/ej2-quickstart-webpack) seed repository.
```
Comment (Yvone-Atieno) [inline ej2-javascript/datepicker/ts/getting-started.md:14]: Check JavaScript DatePicker link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: ej2-javascript/datepicker/ts/getting-started.md (hunk 1 above)
Comment (berylouma) [reply ej2-javascript/datepicker/ts/getting-started.md]: Corrected

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Pull request merged**